### PR TITLE
Fix: Addons details add status shedule

### LINKED ISSUE #264: [Feature] Show the addon detail status.
Now if an addon get stuck in `enabling` phase because of some reasons,  there aren't any ways for user to know what happened.  This [pr](https://github.com/oam-dev/kubevela/pull/2910) set a field as `DetailWorkflowInfo` which contains the addon related application `detailWorkflowInfo`. And this field will always have the the reason of application hanging.
eg:
    
{
"name": "fluxcd",
"phase": "enabling",
"args": null,
"detailWorkflowInfo": "[{"id":"498cywf5sl","name":"deploy-control-plane","type":"apply-application","phase":"failed","message":"step output: step components: step 1: run step(provider=oam,do=component-apply): DispatchStandardWorkload: cannot apply manifest, name: \"webhook-receiver\" apiVersion: \"v1\" kind: \"Service\": cannot create object: services \"webhook-receiver\" is forbidden: unable to create new content in namespace flux-system because it is being terminated","reason":"Execute","firstExecuteTime":"2021-12-10T10:23:46Z","lastExecuteTime":"2021-12-10T10:23:46Z"}]"
}

### PR DESCRIPTION
**Pull request by wangbow**
Signed-off-by: wb-wb665667 <wb-wb665667@alibaba-inc.com>

Fixes #264 

I have:

-  [x] Mars Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
-  [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary.
-  [x] Run `yarn lint` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### Special notes for your reviewer

image

### CODE REVIEW

File: src/pages/Addons/components/detail/index.tsx [hunk 1]
```diff
@@ -201,12 +243,30 @@ class AddonDetailDialog extends React.Component<Props, State> {
...
           <Loading visible={loading} style={{ width: '100%' }}>
+            <If condition={addonsStatus}>
+              <Message
+                type={getAppStatusShowType(addonsStatus?.status)}
+                size="medium"
+                style={{ padding: '8px', marginBottom: '10px' }}
+              >
+                <Translation>{`Addon is ${addonsStatus?.status || 'Initing'}`}</Translation>{' '}
+                <a onClick={() => this.updateStatusShow(true)}>
+                  <Translation>Check the shedule</Translation>
```
Comment (barnettZQG) [inline src/pages/Addons/components/detail/index.tsx:254]: Check the shedule ？

File: src/pages/Addons/components/detail/index.tsx [hunk 2]
```diff
@@ -308,6 +368,33 @@ class AddonDetailDialog extends React.Component<Props, State> {
...
+              <Button type="primary" onClick={this.loadAddonStatus}>
+                <Translation>Refresh</Translation>
+              </Button>
+            </div>
+          }
+          height="auto"
+          footerAlign="center"
+        >
+          <Loading visible={statusLoading} style={{ width: '100%' }}>
+            <Status applicationStatus={addonsStatus} />
```
Comment (barnettZQG) [inline src/pages/Addons/components/detail/index.tsx:395]: Include Dialog in  Status component?

File: src/extends/Group/index.tsx [hunk 3]
```diff
@@ -18,6 +18,10 @@ type Props = {
   field?: Field;
   jsonKey?: string;
   propertyValue?: any;
+  alwaysShow?: boolean;
+  disableAddon?: boolean;
+  titleLeftGrid?: string;
```
Comment (barnettZQG) [inline src/extends/Group/index.tsx:23]: Why add titleLeftGrid and titleRightGrid?

File: src/pages/Addons/components/detail/index.tsx (hunk 1 above)
Comment (wangbow) [reply src/pages/Addons/components/detail/index.tsx]: done

File: src/pages/Addons/components/detail/index.tsx (hunk 2 above)
Comment (wangbow) [reply src/pages/Addons/components/detail/index.tsx]: Extract common components to deal with different status details dialog

File: src/extends/Group/index.tsx (hunk 3 above)
Comment (wangbow) [reply src/extends/Group/index.tsx]: Remove it .done

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**